app/controllers/granules_controller.rb

```diff
@@ -6,6 +6,45 @@ class GranulesController < ApplicationController
   def show
   end
 
+  # Pulls a new random granule and associates with the specified collection id.
```

Comment:
Clarify that this isn't changing CMR but rather working with records, reviews ,etc.